File: js/04-heranca/Classes/Pessoa/Fisica.js
import { Pessoa } from "./Pessoa.js";

class Fisica extends Pessoa{
    constructor(id, mainName, fone, city, cpf){
        super(id, mainName, fone, city);
        this.cpf = cpf;
        this.company = null;
    }

    setCompany(company){
        this.company = company;
    }
}

export { Fisica }

File: py/04-heranca/Classes/Pessoa/Fisica.py
from Classes.Pessoa.Pessoa import Pessoa
from Classes.Cidade.Cidade import Cidade

class Fisica(Pessoa):

    def __init__(self, name, fone, city, cpf):
        super().__init__(id, name, fone, city)
        self.cpf = cpf
        self.company = None

    def setCompany(self, company):
        self.company = company